resources/service-monitor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: prometheus-operator
+    release: prometheus-kubeaddons
+  name: cassandra-cluster-monitor
+spec:
+  endpoints:
+  - interval: 30s
```

Comment:
This YAML file is not properly indented, please check out: https://github.com/mesosphere/kudo-cassandra-operator/blob/05b09e8127c02e179435975c0b127e432fde134e/README.md#running-static-code-analyzers